### DISCUSSION TITLE
Docusaurus 自动提交 url 到搜索引擎 - Alan|前端博客

### DISCUSSION THREAD

**Discussion by giscus** [3Alan/site#9] [Announcements]
# Docusaurus 自动提交 url 到搜索引擎 - Alan|前端博客

通过 Github Actions 自动提交 Docusaurus url 到搜索引擎（百度、必应、谷歌）

https://www.alanwang.site/posts/%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA%E7%B3%BB%E5%88%97/Docusaurus-%E6%90%9C%E7%B4%A2%E5%BC%95%E6%93%8Eurl%E6%8F%90%E4%BA%A4

**Comment by Fionn96** [comment]
大佬，根据您的教程新建 workflows 自动提交 url 到搜索引擎，产生了如下报错：
![](https://s1.vika.cn/space/2022/11/09/4a5f1923c16048f59a615e89699207ba)
提交不成功，请问该如何解决？

**Reply by Fionn96** [reply]
这是我的环境变量设置：
![](https://s1.vika.cn/space/2022/11/09/e2bdacccd766474b98329b1a0b5cad3d)

**Reply by 3Alan** [reply]
你是在 github 仓库的 secert 里面配置的环境变量吗？本地的 .env 文件应该读取不到的

**Reply by Fionn96** [reply]
上传了仓库的，只是我在本地截图而已。我还试过把那些变量 Actions secrets 直接写到 workflows 里也不行

**Reply by 3Alan** [reply]
我的项目运行是正常的，你提供一下你的仓库地址，不然看不到代码我也没法排错

**Reply by Fionn96** [reply]
[https://github.com/Fionn96/my-site](https://github.com/Fionn96/my-site)
因为还在制作所以之前没公开，现在我公开了，麻烦您看一下😂

**Reply by 3Alan** [reply]
看了下你最近一次的失败信息
```
Quota exceeded for quota metric 'Publish requests' and limit 'Publish requests per day' of service 'indexing.googleapis.com' for consumer
```
你谷歌今天的配额已经超出了，另外建议你调试的时候想把 count 设置为 1，等调通了在修改为正常值
